### ISSUE TITLE
Exercises?

### ISSUE THREAD

**Issue by bgreenwell** [koalaverse/homlr-2ed#2]
@bradleyboehmke should we take this opportunity to add end of chapter exercises? I think this would be a hefty improvement to the 1st edition.

**Comment by bradleyboehmke** [comment]
Yeah, certainly. I have a bunch of exercises from classroom material so as we build out each chapter I will incorporate my pre-existing exercises and then we can modify as necessary.
